webpack.config.js

```diff
@@ -0,0 +1,56 @@
+const dotenv = require('dotenv');
```

Comment:
Could we use es6 syntax here, and the rest of the lines of code up to line 7

Reply:
@nkalyesubula  I don't think that's absolutely necessary as it's a config file.

Reply:
@nkalyesubula @salviosage I have moved babel config to an independent babel.config js file and migrated the webpack file to es6. Thanks

Reply:
@salviosage was right. for this file it's not necessary